src/controller_examples/rabbitmq_controller/exec/reconciler.rs

```diff
@@ -298,21 +298,19 @@ pub fn reconcile_core(rabbitmq: &RabbitmqCluster, resp_o: Option<Response<EmptyT
                     if found_stateful_set.is_ok(){
                         let mut new_stateful_set = found_stateful_set.unwrap();
```

Comment:
The term `new_stateful_set` is a bit confusing. Can we term it as `found_stateful_set` and the previous `found_stateful_set` becomes `get_sts_result`?

Reply:
I also don't know why it was named this way.